pkg/controller/service_control.go

```diff
@@ -39,6 +41,7 @@ type ServiceControlInterface interface {
 	CreateService(runtime.Object, *corev1.Service) error
 	UpdateService(runtime.Object, *corev1.Service) (*corev1.Service, error)
 	DeleteService(runtime.Object, *corev1.Service) error
+	SyncComponentService(runtime.Object, *corev1.Service, *corev1.Service, bool) (*corev1.Service, error)
```

Comment:
it's better to add some comments or naming these args.

Reply:
added the naming